src/Storages/Statistics/tests/gtest_stats.cpp

```diff
@@ -39,6 +43,51 @@ TEST(Statistics, TDigestLessThan)
 
     std::reverse(data.begin(), data.end());
     test_less_than(data, {-1, 1e9, 50000.0, 3000.0, 30.0}, {0, 100000, 50000, 3000, 30}, {0, 0, 0.001, 0.001, 0.001});
+}
+
+using Fields = std::vector<Field>;
```

Comment:
The test l. 48-92 belongs to the "Statistics" tests (l. 71) but in fact, it tests `convertFieldToType` and is unrelated to statistics. I would say what it tests is already covered by existing tests of `convertFieldToType`. Suggest to remove the the test.

Reply:
I did not find the related unit test cases in `gtest_convertFieldToType`, I moved the test cases to `gtest_convertFieldToType`.

Reply:
That's fine, thanks.